### ISSUE TITLE
Support for SKR Mini E3 V2.0?

### ISSUE THREAD

**Issue by 2000game** [Jbchow/Marlin-Ender-5-Plus#2]
Hello,
will this Firmware work with the SKR V2.0?
Or are you planning on supporting it in the near future?

**Comment by Jbchow** [comment]
I do not believe it will work with the SKR boards easily. The reason I stopped updating this branch is because I purchased a SKR Pro 1.4 and it was easier to start over with base marlin and modify it for my use than to make this branch work on the SKR. I am sure it can be done but I wasn't going to do it. I have since replaced almost every part on the printer except the frame and the main belts. I am not sure my config is useful for anyone except me at the moment.
